axelor-redmine/src/main/resources/views/TeamTask.xml

```diff
@@ -58,5 +59,16 @@
 		<view-param name="popup" value="true" />
 		<domain>self.teamTask.id = :id</domain>
 	</action-view>
+	
+	<action-attrs id="redmine-action-team-task-attrs-invoice-type-on-change" name="action-team-task-attrs-invoice-type-on-change">
```

Comment:
No need to override this action now as this change is now have to made on aos-dev